lib/ui/form/widget/form_survey_question_page.dart

```diff
@@ -47,4 +51,17 @@ class FormSurveyQuestionPage extends StatelessWidget {
         question.text,
         style: Theme.of(context).textTheme.titleLarge,
       );
+
+  Widget _buildAnswer(BuildContext context) {
+    final displayType = question.displayType;
+    switch (displayType) {
+      case DisplayType.intro:
+      case DisplayType.outro:
+        return Container();
```

Comment:
The API returns the list of question types with the first item as `intro`. Does that mean there is an empty page right after the first `detail page`?

Reply:
@luongvo The text will show for the `intro` item, but for the answer it will empty 🚀 

![Screen Shot 2023-04-24 at 09 31 45](https://user-images.githubusercontent.com/18277915/233886584-31a68eab-dce8-4ee0-960f-acb28c0b6ed3.png)

Reply:
@Tuubz Hmm, from my understanding, the `intro` type is the first survey detail page already, so we should skip showing it, or show it as the first detail page.

Reply:
Hmmm, but what about the `outro` then? If we keep `outro`, then it makes sense to keep `intro` too 💭

Reply:
It seems like you're right 👍 . 

According to this [design flow:](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=38-2124&scaling=scale-down), I don't see any intro or outro, will skip them 🚀

Reply:
@Tuubz Nice question. The `outro` is actually this page. I think we can bind that page from the question data or hardcoded the page content as a static page 🤷‍♂️ 

<img width="204" alt="image" src="https://user-images.githubusercontent.com/16315358/233889525-dac76051-1393-45cc-bd9d-af2c98ff3e17.png">

<img width="682" alt="image" src="https://user-images.githubusercontent.com/16315358/233889636-08596e99-1d9a-4dbc-a891-f3d57fabd245.png">